process/charter.html

```diff
@@ -178,13 +178,16 @@ <h2>3. <a id="creation" name="creation">Charter Creation</a></h2>
 
 <h3 id="horizontal-review">3.1 Horizontal Review</h3>
 
-	<p>Once there is agreement on a draft charter, ideally among Team Contact(s), candidate chairs(s), relevant interest groups, and interested community members, seek review from the Strategy Team and horizontal reviewers by:
-	<ol><li>Opening a <a href="https://github.com/w3c/strategy/issues/new?assignees=&labels=Evaluation%3A+untriaged&template=04-Chartering.md&title=">Chartering issue</a> in the <a href="https://github.com/w3c/strategy/issues">Strategy Repo</a>,</li>
+	<p>Once there is agreement on a draft charter, ideally among Team Contact(s), candidate chairs(s), relevant interest groups, and interested community members, seek review from the Strategy Team and horizontal reviewers by:</p>
+	<ol>
+  <li>Moving/Copying the draft charter into the <a href="https://github.com/w3c/charter-drafts/">charter-drafts Repo</a>, naming it using the convention [YYYY]/[shortname-name]-[group-type].html .</li>
+  <li>Opening a <a href="https://github.com/w3c/strategy/issues/new?assignees=&labels=Evaluation%3A+untriaged&template=04-Chartering.md&title=">Chartering issue</a> in the <a href="https://github.com/w3c/strategy/issues">Strategy Repo</a>,</li>
```

Comment:
These issues should be filed in charter-drafts, not strategy. The point of having a single repo is to have a _single place_ where everyone knows the charter drafting discussion is happening. Leaving the draft in one repo and the issues in another is not helping.

Also, mixing up strategy discussions with charter drafting issues is also not helpful, imho. Charter drafting is not strategy by a long shot. Let's keep them separate so people can discuss strategy and new frontiers in the strategy repo, and discuss charter drafting in the charter drafting repo.